src/main/java/seedu/ui/Ui.java

```diff
@@ -32,7 +32,7 @@ public Ui() {
      * Prints introductory message when Fridget runs.
      */
     public void printIntroduction() {
-        String logo = "\n"
+        String logo = "\n\n\n\n"
```

Comment:
why here got extra \n? (just checking)

Reply:
i figured after using the .jar file, the logo too near the command line that launched the file not a pretty sight.

Reply:
Oh I see